freemocap/gui/qt/widgets/camera_controller_group_box.py

```diff
@@ -65,7 +67,13 @@ def _create_videos_will_save_to_layout(self):
         self._recording_path_label = QLabel(f"{self.get_new_recording_path()}")
         self._recording_path_label.setStyleSheet("font-family: monospace; font-size: 10px;")
         hbox.addWidget(self._recording_path_label)
-        return hbox
+
+        vbox.addLayout(hbox)
+
+        lag_note_label = QLabel("NOTE: Use fewer cameras or decrease resolution if you are experiencing lag")
```

Comment:
```suggestion
        lag_note_label = QLabel("NOTE: If you experience lag in your camera views, decrease the resolution and/or use fewer cameras.\nWe are working on a fix which should be done 'soon' (written: 2023-04-05)")
```

Changed my mind again on this text, please make sure that newline `\n` works properly in the GUI